### ISSUE TITLE
"The 'cache' field on 'RequestInitializerDict' is not implemented" in live cloudflare worker

### ISSUE THREAD

**Issue by tobbbe** [upstash/upstash-redis#338]
Getting `The 'cache' field on 'RequestInitializerDict' is not implemented.` when trying to run this code in a worker (live). It does work fine in locally though. Whats wrong? 🤔

```
import { Redis } from '@upstash/redis/cloudflare'

const cfModule = {
  async fetch(request: Request, env: Env) {
    const asd = await Redis.fromEnv(env).lrange('asd', 0, 100)
    return new Response(JSON.stringify(asd), {
      headers: { 'Content-Type': 'application/json' },
    })
  },
}
```

**Comment by chronark** [comment]
Thanks, I'll fix it. In the meantime, can you try  an older version, 1.19.1 should work

**Comment by tobbbe** [comment]
1.19.1 works! Thanks! 🙌

**Comment by donchev7** [comment]
I can confirm that this bug exists only on **1.20.2** . After reverting dependabot PR to 1.20.1 it works as expected.

**Comment by donchev7** [comment]
I believe this [commit](https://github.com/upstash/upstash-redis/commit/4b7b3f60a40348c66ed3945e810d1bfbd370ea83) introduced the "bug". 

However, I believe its only relevant on cloudflare since they do not support the cache field in fetch but other platforms do.
We might track this [issue](https://github.com/cloudflare/workers-sdk/issues/3005) and see if they add support for the cache field....

**Comment by chronark** [comment]
Sorry about that, our ci broke and didn't catch this before releasing
Fixed it in `v1.20.3`
